.github/workflows/ci.yml

```diff
@@ -15,6 +15,9 @@ on:
   pull_request:
   # Allow manual runs through the web UI
   workflow_dispatch:
+  # Trigger on completion of the scheduled_builds.yml file (only on main)
+  workflow_run:
+    workflows: [Scheduled build triggerer]
```

Comment:
Ok, here's a question I can't quite decide what option is the least-worst.

As currently implemented here, this triggers when the `scheduled_builds` workflow finishes. This works well for this repo, as we now only have one `scheduled` trigger in that workflow, which triggers everything. However, I want to implement this in the template and I am wondering if having separate `schedule` triggers in the `ci.yml` and `cron.yml` would be more accessible for projects which do not use release branches (almost all of them) and we should just suck it up in sunpy/sunpy and sunpy/ndcube and have three separate, duplicated `schedule` triggers.

Reply:
Will those projects eventually move to release branches? 

If no, ndcube/sunpy can have repeated triggers.

Reply:
> Will those projects eventually move to release branches?

I mean maybe eventually. Only if they support more than one release at a time.

The alternative is that we mirror what we have here into the package template, but the `scheduled_builds` workflow is a no-op which is just used to trigger the other workflows?

Reply:
No op sounds ok to me